Riot/Modules/Room/RoomViewController.m

```diff
@@ -4796,28 +5030,34 @@ - (void)roomInputToolbarViewDidOpenActionMenu:(RoomInputToolbarView*)toolbarView
 
 - (void)roomInputToolbarView:(RoomInputToolbarView *)toolbarView sendAttributedTextMessage:(NSAttributedString *)attributedTextMessage
 {
-    BOOL isMessageAHandledCommand = NO;
-    // "/me" command is supported with Pills in RoomDataSource.
-    if (![attributedTextMessage.string hasPrefix:kMXKSlashCmdEmote])
-    {
-        // Other commands currently work with identifiers (e.g. ban, invite, op, etc).
-        NSString *message;
-        if (@available(iOS 15.0, *))
+    // Create or invite again the left member before sending the message in case of a discussion (direct chat)
+    MXWeakify(self);
+    [self createOrRestoreDiscussionIfNeeded:^(BOOL success) {
```

Comment:
It seems we don't check the `success` value before actually sending message. It may be a good idea to check it and display an error if not successful.

Reply:
I added the missing `success` check in the last commit.